### PR TITLE
chore: cli tweaks

### PR DESCRIPTION
**Pull request by colomolo**

### CODE REVIEW

File: packages/create-contentful-app/package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "create-contentful-app",
   "version": "1.1.0-alpha.3",
   "description": "A template for building Contentful Apps",
+  "keywords": ["Contentful", "create-contentful-app", "CCA"],
```
Comment (shikaan) [inline packages/create-contentful-app/package.json:5]: @stefanjudis Do you happen to have a list of good keywords we should be using here?

File: packages/contentful--create-contentful-app/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ This project makes it easy to bootstrap [Contentful Apps](https://www.contentful
 ## Requirements
 
 - Node.js v14 or v16
-- NPM v6
+- npm v6
```
Comment (shikaan) [inline packages/contentful--create-contentful-app/README.md:8]: I think this was meant to be changed in the --help too. We also need to review all the links there, I fixed one yesterday, not sure about the rest

Comment (colomolo) [reply packages/contentful--create-contentful-app/README.md:8]: yep, I changed it the help command too. Which links do you mean?

File: packages/contentful--create-contentful-app/src/logger.ts [hunk 3]
```diff
@@ -16,13 +16,13 @@ export function highlight(str: string) {
...
 }
 
 export function success(str: string) {
-  return chalk.cyan(str);
+  return chalk.greenBright(str);
 }
 
 export function code(str: string) {
-  return chalk.dim(str);
+  return chalk.bold(str);
```
Comment (colomolo) [inline packages/contentful--create-contentful-app/src/logger.ts:27]: Color changes were approved by @fabe

File: packages/contentful--create-contentful-app/README.md (hunk 2 above)
Comment (shikaan) [reply packages/contentful--create-contentful-app/README.md:8]: https://github.com/contentful/create-contentful-app/blob/f338b5a609b8bdba1d9095a399abc8974025a723/packages/contentful--create-contentful-app/src/index.ts#L132

This is still uppercase ☝️  Also, `Yarn -> yarn` maybe

Links are already updated, you are right :)

File: packages/create-contentful-app/package.json (hunk 1 above)
Comment (stefanjudis) [reply packages/create-contentful-app/package.json:5]: I'd drop `CCA`. Primarily `contentful` (I'd go with a lower `c`) is important I'd say. :)

**Review by stefanjudis — APPROVED** [review]
🎊 🚢

**Review by shikaan — APPROVED** [review]

**Pull request merged**